composer.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "php-di/zend-expressive-bridge",
+  "description": "PHP-DI integration in Zend Expressive",
+  "license": "MIT",
+  "type": "library",
+  "require": {
+    "php": "@stable",
+    "php-di/php-di": "@stable",
+    "php-di/invoker": "@stable",
+    "zendframework/zend-expressive": "@stable",
+    "zendframework/zend-expressive-helpers": "@stable"
```

Comment:
I think it's better to restrict to major versions to avoid being affected by a BC break in any of those packages.

Random example: the class `Zend\Expressive\Container\ApplicationFactory` could very well be removed in a new major version.